salt/utils/rsax931.py

```diff
@@ -30,6 +31,24 @@ def _find_libcrypto():
     """
     if sys.platform.startswith("win"):
         lib = str("libeay32")
```

Comment:
Now that @s0undt3ch mentions it, here as well.

Reply:
Ahh, found it. There was a comment here previously noting that Windows required the `str` call. Is that still the case, or was that an old artifact of Python 2?

Reply:
add the comment back in for windows and removed the str cast for macOS as it's not required.

Reply:
@twangboy do we still need this `str()` cast? I believe this was only necessary for Py2, right?

Reply:
I validated that on windows salt with py3 the cast is not needed. @s0undt3ch how do you want to proceed?

Reply:
Nuke it too :explosion: :)

Reply:
Sorry, Please.

Reply:
@s0undt3ch changed and ready to go :)